README.md

```diff
@@ -16,6 +16,42 @@ _p_**REST** (**P**_ostgreSQL_ **REST**), simplify and accelerate development, in
 
 There is PostgREST written in Haskell, but keeping Haskell software in production is not an easy job. With this need pREST was born. [Read more](https://github.com/prest/prest/issues/41).
 
+## Development usage
+
+For dependencies installation
+
+```sh
+go mod download
+```
+
+Building a local version
+
+```sh
+go build -ldflags "-s -w" -o prestd cmd/prestd/main.go
```

Comment:
Since it is a test, it would be interesting to use the docker images (simplifying the test of who is arriving in the project)

what is your opinion?

Reply:
There is an option for a development mode, i have checked the docker images but not have all the work for a local execution.